dynawo/sources/Simulation/DYNSimulation.cpp

```diff
@@ -281,6 +281,9 @@ void
 Simulation::configureConstraintsOutputs() {
   // Constraints settings
   if (jobEntry_->getOutputsEntry()->getConstraintsEntry()) {
+    stringstream pid_string;
```

Comment:
Maybe we can remove the timeline from solvers since it is not used?

Reply:
it is used at least by solverSIM in solveWithStepRecalculation

Reply:
OK then there should be an update on the solver side right?